src/js/containers/Dashboard/DashboardPostPage/index.js

```diff
@@ -29,18 +30,27 @@ import {
   ErrorNotification,
   PostList,
   PostListItemDetail,
-  SectionForm,
-  MarqueeForm
+  SectionLayoutForm,
+  MarqueeForm,
+  BoxLayoutForm
 } from 'grommet-cms/components';
 import {
   toggleSectionForm,
   postSectionFormInput,
   postSectionFormReset,
   postSectionSetToastMessage,
-  postSectionClearToastMessage
+  postSectionClearToastMessage,
+  postBoxLayoutFormReset,
+  postBoxLayoutFormInput,
+  toggleBoxLayoutForm,
+  postToggleAdvancedLayout
 } from './actions';
-import parseSubmission from './utils';
 import { debounce } from 'grommet-cms/utils';
+import {
```

Comment:
This container is getting giant.  If we need to add anything else I think it we should make another container